core/app/models/spree/reimbursement.rb

```diff
@@ -148,6 +150,14 @@ def return_all
 
     private
 
+    def recalculate_order
+      order.recalculate
+    end
+
+    def calculate_total
+      self.total ||= calculated_total
```

Comment:
maybe self is redundant here?

Reply:
Removing it assign the value to a new variable.